pydoctor/templatewriter/pages/__init__.py

```diff
@@ -279,6 +279,11 @@ def __init__(self, ob, docgetter=None):
                 self.baselists.append((baselist, attrs))
         self.overridenInCount = 0
 
+    def children(self):
+        return sorted((o for o in self.ob.contents.values()
+                       if o.name != '__init__' and o.isVisible),
```

Comment:
I think this will hide the `__init__` methods and `__init__` modules. As you can in the docs generated for this PR: [here](https://pydoctor--379.org.readthedocs.build/en/379/api/pydoctor.templatewriter.writer.TemplateWriter.html) for instance, the `__init__` method do not show up...

Let's note that we are discussing about how to display of constructors [here](https://github.com/twisted/pydoctor/issues/305). And I think the PR #360 will solve the issue with `__init__` module everywhere.

About the sorting order, @mthuurne will correct me if I'm wrong, but I think it's a design decision (probably discutable) to list members in the same order they have been defined in the code. So maybe this should be an option like `--sort-members` for instance. Also sorting by kind would be useful maybe?

Reply:
Once #360 lands, it will no longer be necessary to filter out `__init__` modules, since `__init__.py` will be treated as the source file for the package rather than a module within the package.

I don't know whether using the source order was a design decision or just a result of the most straightforward implementation. For most projects, source order is either pure chaos or some logic that only the developer understands, so sorting the names would benefit the reader, in my opinion.

I'll post a separate reply at the top level to discuss the sort order, since that's more of a requirements discussion than a code discussion.

Reply:
Actually, this code is inside `ClassPage`, so it would only be filtering out `__init__()` methods. If there is a reason why you wouldn't want to see the `__init__()` methods listed, please document it in #305. But for the time being, we'd lose information by simply dropping them unconditionally for all projects.

Reply:
This method was blindly copied from `PackagePage` where `__init__` is also filtered, although I guess that `__init__` refers to `__init__.py` there, not the `__init__` method, right? If that is the case, then indeed it should not be filtered.